U-4813 Link documentation and examples, update README (#33)
BetterStackHQ/terraform-provider-logtail: README.md

> ## Example Usage

Reviewer: ```suggestion
## Example usage
```

> See [`/examples` directory](./examples) for multiple executable configurations.

Reviewer: ```suggestion
See [`/examples` directory](./examples) for multiple ready-to-use examples.
```

## Documentation

> See [Better Stack Telemetry API docs](https://betterstack.com/docs/logs/api/getting-started/) to obtain API Token and get the complete list of parameter options.

Reviewer: ```suggestion
See [Better Stack Telemetry API docs](https://betterstack.com/docs/logs/api/getting-started/) to obtain API token and get the complete list of parameter options.
```
